src/tic_tac_toe/player.clj

```diff
@@ -1,12 +1,5 @@
-(ns tic-tac-toe.player
-  (:require [tic-tac-toe.user-interface :as ui]))
+(ns tic-tac-toe.player)
 
 (defprotocol Player
-  (pick-move [this ui]))
-
-(defrecord Human []
-  Player
-  (pick-move [this ui] (ui/get-input ui)))
-
-(defn create-player []
-  (map->Human {}))
+  (pick-move [this game-ui board])
+  (validation [this move]))
```

Comment:
I feel like validation should not up to the player, and it's kind of a responsibility of a referee

Reply:
It's ok to validate a computer's move, it's not up to the referee to make their decision based on the player type, that would make it a pretty partial judge and the professional TTT leagues would have that referee banned from participating

Reply:
OK, brain was overcomplicating this.  Changed it to a [few functions](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-f014d0e7db79dee6fc42af72f74d5325R5) on ref which handle human and computer types of input.